### PR TITLE
chore(mutants): replace deprecated humbug with infection

### PR DESCRIPTION
**Pull request by abacaphiliac**

### CODE REVIEW

**Comment by abacaphiliac** [comment]
@snapshotpl do you mind taking a look at this?

it's related to #57 ... the 74snapshot fails on lowest dependency versions which i'll clear up in a subsequent commit:
* commit: https://github.com/enlitepro/enlite-monolog/compare/master...abacaphiliac:upgrade-dev-deps?expand=1
* build: https://travis-ci.org/abacaphiliac/enlite-monolog/builds/628185995

that build fails because humbug is old and deprecated, hence this first PR.

File: composer.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
         "lint:syntax": "vendor/bin/parallel-lint --exclude vendor --exclude test/ZF2/vendor --exclude test/ZF3/vendor .",
         "lint:syntax-php53": "vendor/bin/parallel-lint --exclude src/EnliteMonolog/Service/MonologServiceAwareTrait.php config/ src/ test/",
         "test:coverage": "vendor/bin/phpunit --coverage-text",
-        "test:mutations": "vendor/bin/humbug && cat humbuglog.txt",
+        "test:mutations": "vendor/bin/infection && cat humbuglog.txt",
```
Comment (snapshotpl) [inline composer.json]: `humbuglog.txt` is not up to date now

Comment (abacaphiliac) [reply composer.json]: good catch. i've replaced all occurrences of `humbuglog.txt`.

**Comment by snapshotpl** [comment]
@abacaphiliac :+1:

**Pull request merged**